docs/docsite/rst/user_guide/windows_faq.rst

```diff
@@ -36,8 +36,7 @@ Can Ansible run on Windows?
 No, Ansible cannot run on a Windows host and can only manage Windows hosts, but
 Ansible can be run under the Windows Subsystem for Linux (WSL).
 
-.. note:: The Windows Subsystem for Linux is not supported by Microsoft or
-    Ansible and should not be used for production systems. 
+.. note:: The Windows Subsystem for Linux is not supported by Ansible. 
```

Comment:
I would keep the `and should not be used for production systems.`. It's ok for people to use it as a dev environment and it works perfectly well we just don't recommend/support it in an actual production like setting.

Reply:
I understand. Thank you.